.flake8

```diff
@@ -1,7 +1,7 @@
 [flake8]
 select = B,B9,C,E,F,N,W
 ignore = E203,E501,W503,N816
-max-line-length = 80
+max-line-length = 88
```

Comment:
Just curious: is this 88 a standard number?

Seems that PEP8 recommends 80:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
It's a great question that I'm willing to think about more closely.

Generally, it seems people like to either:
a.) cite PEP-8 as the gold standard
or
b.) cite Raymond Hettinger's "[Beyond PEP-8](https://www.youtube.com/watch?v=wf-BqAjZb8M)" guidelines

In the latter, he says 10% over the default 79-80 is acceptable some of the time, but you should on average go for  80. 

I set it to 88 because I wasn't sure if flake8 checks line length with or without the tolerance in mind. Let me verify. If it's already 80+(.1*80) then we should change it back to 80 XD

Reply:
One more thing on this:
Black uses 88 and has the rationale here: https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length

Reply:
Ok I dug in here, and the original setting of 80 was actually fine because we were already passing: `ignore = E501` and using `flake8-bugbear`'s line length error instead: `B950`. This error builds in the tolerance I was mentioning.